README.md

```diff
@@ -35,22 +71,34 @@ n 12.0.0
 npm --version
 ```
 
-## Installation & Usage
+#### Main installation section
```

Comment:
Switch back to explaining with codeblock, remove configuring `bin/www`, and possibly move the Heroku part to it's own section

Reply:
Alright, but it's not actually heroku, but it's for all the platforms which use external `.env` config.